GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -53,15 +88,15 @@ extension FoodSearchViewController: UISearchBarDelegate {
     
     func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
         print("👍 pressed")
-        if let text = searchBar.text {
-            if !text.isEmpty {
-                print("Text: \(text)")
-                if let url = setURL(with: text) {
+        if let inputText = searchBar.text {
```

Comment:
This is another good place to use a `guard` so you don't have the doubly nested `if` statements.

Reply:
I was of two minds about this at first (guards vs if let). Looking at it again, using guards will make this more readable. Changed.